### PR TITLE
Add Botify data source

### PR DESCRIPTION
**Pull request by eisenbergrobin**
This adds the Botify organization and Botify API datasource to GDS sources.

### CODE REVIEW

**Comment by eisenbergrobin** [comment]
I signed it!

File: sources/botify_api.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "id": "BOTIFY API",
```
Comment (pfrisella) [inline sources/botify_api.json]: A more appropriate name is likely just Botify. Consider that the purpose of this is for users to be able to find connectors that use this source. Most often users are not looking for an API but the source of data.

So I recommend the following:
- Change id to BOTIFY
- Change name to Botify
- Change filename to botify.json

Comment (pfrisella) [reply sources/botify_api.json]: Can you also add the SEO category to the categories.json file since I think yours is the first entry with that category.

**Comment by eisenbergrobin** [comment]
@pfrisella Thanks, I've taken your comments into account and changed my PR accordingly.

**Pull request merged**